vanilla-framework/vanilla-framework
# Resize delete icon

## Done
- Updated correct width on SVG
- Center aligned inside it's container

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/examples/patterns/icons/icons-light/
- Inspect `p-icon--delete` and make sure it's in the center

## Details
Fixes https://github.com/vanilla-framework/vanilla-framework/issues/2085

## Screenshots
![screenshot 2019-01-10 at 14 53 02](https://user-images.githubusercontent.com/17748020/50976477-f7dd7f00-14e7-11e9-9122-794a5e3babf1.png)
